addons/mail/models/mail_message.py

```diff
@@ -946,8 +953,6 @@ def _message_format(self, format_reply=True, msg_vals=None, for_current_user=Fal
                     'parentMessage': {...}, # formatted message that this message is a reply to. Only present if format_reply is True
                 }
         """
-        self.check_access_rule("read")
```

Comment:
Note to reviewer:
It is completely useless, kept from when `_message_format` was still a public method.
This is adding extra queries when fixing inbox flow (`with_user` checking extra ACL).